image/tarexport/load.go

```diff
@@ -396,8 +397,16 @@ func checkValidParent(img, parent *image.Image) bool {
 	if len(img.History)-len(parent.History) != 1 {
 		return false
 	}
-	for i, h := range parent.History {
-		if !h.Equal(img.History[i]) {
+	for i, hP := range parent.History {
+		hC := img.History[i]
+		if (hP.Created == nil) != (hC.Created == nil) {
+			return false
+		}
```

Comment:
In the interest of highlighting the bits I'm the least sure about, this is one I _think_ I've got right, but am not positive.